sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -1607,10 +1608,6 @@ class MultiFileCloudParquetPartitionReader(
       hostBuffer: HostMemoryBuffer,
       dataSize: Long,
       fileName: String): Option[ColumnarBatch] = {
-    if (dataSize == 0) {
-      // shouldn't ever get here
-      None
```

Comment:
Possibly, I deleted it since it is dead code. I did not want to change the behavior

Reply:
I don't think we should just delete it because it's dead code if it's clear the intent of the original author was not to create some dead code.  I'm cool with checking this in with the behavior unchanged, but there should be a followup to investigate this.